pkg/environment/environment.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2020 IBM Corp.
 // SPDX-License-Identifier: Apache-2.0
 
-package utils
+package environment
 
```

Comment:
Move the functions that do not return env value (like SetIfNotSet, etc.) to utils.go